util/src/main/java/tk/zielony/randomdata/util/StringVersionGenerator.java

```diff
@@ -0,0 +1,78 @@
+package tk.zielony.randomdata.util;
+
+import android.support.annotation.Nullable;
+
+import java.util.Random;
+
+import tk.zielony.randomdata.DataContext;
+import tk.zielony.randomdata.Generator;
+
+/**
+ * Created by kirinpatel on 12/1/17.
+ */
+
+public class StringVersionGenerator extends Generator<String> {
```

Comment:
This generator could be more generic with options to specify a custom separator and any number of sections.